### ISSUE TITLE
New World: Aeternum (1063730)

### ISSUE THREAD

**Issue by masonnl** [ValveSoftware/Proton#5772]
# Compatibility Report
- Name of the game with compatibility issues: **New World**
- Steam AppID of the game: `1063730`

## System Information
- GPU: GTX 3060
- Driver/LLVM version: nvidia: 510.60.02 / llvm-config: 13.0.1
- Kernel version: 5.15.32-1-MANJARO
- [System Gist](https://gist.github.com/masonnl/be147f37ae32045a2d4b784ca4acaaec)
- Proton version: Proton [7.0-1](https://github.com/ValveSoftware/Proton/releases/tag/proton-7.0-1)
- GE version [7-14](https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/GE-Proton7-14)
- Proton logfile cannot be uploaded. It exploded to 280MB when created so instead I took the head of the first 10k lines here: [steam-1063730-head10k.log](https://github.com/ValveSoftware/Proton/files/8502632/steam-1063730-head10k.log)
  - _EDIT: here's my compressed logfile taken from [my comment below](https://github.com/ValveSoftware/Proton/issues/5772#issuecomment-1101524196): [steam-1063730.log.zip](https://github.com/ValveSoftware/Proton/files/8506717/steam-1063730.log.zip)_

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.
- [x] EAC support has been added to the game's directory under `EasyAntiCheat/easyanticheat_x64.so` and connects successfully as shown in the EAC logs generated under `~/.cache/com.epicgames.easyanticheat/271/loader.log`. I have been able to play upwards of 10+ hours in the last few days (with crashes as described below).

## Symptoms
- **[[RESOLVED](https://github.com/ValveSoftware/Proton/issues/5772#issuecomment-1102096272)]** ~Except in the rare occasion when the game actually loads, it will simply crash to desktop while loading into a character (see STR below).~
- Stuttering on various settings (even low) with resolution at 1920x1080
  - Playing at full native resolution of 2560x1440 ran fine and was locked at 60fps (with the same amount of stuttering)

## Steps to Reproduce
Scenario 1:
- Hit Play in Steam
- Log in to character
- Loading screen...
- :boom: back to desktop
  - :warning: Very rarely the game will load in and I can play. I have not modified shader files or compat files. It will sometimes just work :shrug: 

Scenario 2:
- Hit play in Steam
- Create new character
- Play completely through the tutorial zone and get into the main world with other players. This seems to have worked for me 2/2 times, but I obviously don't want to do to that :D The first time I did it, I played for a few hours without issue (other than stutters).

**Comment by gofman** [comment]
I couldn't reproduce the crash (loading with existing character worked just fine for me 2 of 2 times).

Could you please try the following:
1. Run the game with official Proton Experimental (or better Proton Experimental [bleeding-edge] branch: select [bleeding-edge] in Proton Experimental's properties, Beta page). It won't work out of the box due to EAC launcher problem which is hacked around in Proton GE build. But it should work with the attached EasyAntiCheat_x86.dll (that is the .dll from EAC SDK correct for Linux EAC). Unzipped dll should be copied over EasyAntiCheat/EasyAntiCheat_x86.dll in the game directory). This way the game should start with the launcher normally with official Proton.
2.If the crash described above still happens with official Proton, could you please attach full (compressed) log (default log from PROTON_LOG=1 %command%). The log may be big but it is compressed very well.

[EasyAntiCheat_x86.zip](https://github.com/ValveSoftware/Proton/files/8506258/EasyAntiCheat_x86.zip)

**Comment by masonnl** [comment]
Hi @gofman :wave: 

Going forward I won't use GE to troubleshoot just to limit the problem space. Assume Proton Experimental is being used unless specified.

> It won't work out of the box due to EAC launcher problem which is hacked around in Proton GE build.

As far as I know, the official New World client shipped all the correct SDK files for EAC with a fresh install via Steam on Linux. Are you saying that the game didn't package the _linux_ eac dll?

> This way the game should start with the launcher normally with official Proton.

I backed up the eac directory, and placed your attached dll in the eac directory. Launch options: `PROTON_LOG=1 %command%` and using Proton Experimental.
Here's the latest (compressed) crash report: 
[steam-1063730.log.zip](https://github.com/ValveSoftware/Proton/files/8506717/steam-1063730.log.zip)

Thanks for your time!

_EDIT: I haven't used bleeding edge before so I glossed over that in your request. Here's a log using Proton Experimental with bleeding edge specified: 
[steam-1063730-bleeding-edge.log.zip](https://github.com/ValveSoftware/Proton/files/8507084/steam-1063730.log.zip)_

**Comment by blae-s** [comment]
Hi @gofman 

GPU: GTX 1060
Driver/LLVM version: nvidia: 510.60.02 / llvm-config: 13.0.1
Kernel version: 5.17.1-xanmod1-1 

I removed GE-proton prefix, switched to proton experimental bleeding edge and copied EasyAntiCheat/EasyAntiCheat_x86.dll as you suggested, with PROTON_LOG=1 %command% as launch options.

Starting the game and loading a character worked 5 times in a row and i was able to play a little while, then crashed the 6th times i started the game before even being able to load a character 

[steam-1063730.log.zip](https://github.com/ValveSoftware/Proton/files/8506865/steam-1063730.log.zip)

**Comment by gofman** [comment]
Thanks for the logs!

@masonnl Somehow both your log gives no clue even on immediate crash reason (that's apparently not your fault). Can you please attach dmesg (or 'sudo dmesg' if your system restricts dmesg to root only) output dumped right after the game has crashed?

@blae-s Your log does have immediate reason but it suggests some very weird memory corruption reason of which I am unlikely to guess without reproducing.

Are there any hints or more details how any of those crashes can be reproduced? I've got a character which I exited the initial ship wreckage / zombies area, got and completed some first "dueling quest" killing some drowners, and loading in to this character. Is that enough? I wasn't able to reproduce any crashes so far.

**Comment by masonnl** [comment]
Looks to be crashing due to an out of memory dump. Possibly a mem leak or something as it is not an issue with my hardware limitations. [dmesg.dump.log](https://github.com/ValveSoftware/Proton/files/8508244/dmesg.dump.log)

As mentioned, this crash occurs very consistently for me. I am not in the tutorial zone of the game, though which it sounds like you may be? I believe I am logged out in a city.

**Comment by gofman** [comment]
Hm, if I am not missing something obvious in arithmetic the game takes roughly 10GB of RAM when killed, which is not obviously an issue on 16GB machine. Looks like something potentially explored with the host maybe.

Just in case, is it long to get out of tutorial zone to the city and, provided that is not too long and easy to say, how do I quickly get to a city out of tutorial zone?

**Comment by masonnl** [comment]
Probably takes 10 minutes from opening the game. Just run where it points you, fight the dude with the laser-beam sword, spawn on the beach, and follow the main road into the nearest town (hit 'M' to reveal the map; you will spawn in 1 of 3(?) random areas so the town name will vary).

**Comment by masonnl** [comment]
Following up on this with some new discoveries:

I got to thinking about the oom-killer and it made me realized a couple things:

1. The oom-killer was triggered at exactly 10GB with the appropriate kernal-defined leftovers of ~4GB being preserved for the system.
2. This made me realize I do not have a swapfile setup for this system; this will be important later. 
 
To help see if I was indeed hitting some system-defined process memory cap I ran the following command in the background while booting: `free -h -s1` to watch available process memory while loading into New World. As you might expect, available process memory approached 0 and the game process was killed.

I added a swapfile to my system, made an fstab entry, and rebooted to make sure I was repeating the above experiment with a clean slate. Upon loading in, I again saw the available process memory approach 0, but this time the kernal utilized the swapfile for the (seriously egregious) memory usage and I successfully loaded in.

**Comment by masonnl** [comment]
I can confirm that the expanded memory availability resolves the crash I was reporting as part of this compatibility report :tada:

To sum up my findings:

1. Sanity check dmesg to see if there's some system-wide problem that kills the game outside of Proton/Steam. In my case this was an obscene amount of memory use that caused oom-killer to end the NewWorld.exe process. Thank you @gofman for pointing me in this direction.
2. Changing the in-game settings while logged in to a character in New World uses A TON of memory. I resulted to restarting the game to change the visual settings from the main menu screen instead to prevent crashes even after I knew what the root cause was. A swapfile with 8GB is needed on my 16GB system to play on High settings. In game memory usage after 2 hours of gameplay was sitting at <11GB. VRAM was sitting at <6GB. PSA: I had to clear my swap when closing the game so that I could even get things to load upon reopening the game. If others are experiencing crashes related to memory, keep this in mind. If more experienced folks have insight in swappiness fine-tuning, I'd be interested in hearing from you!
4. Adjusting the visual settings did not resolve the stuttering. My settings are as follows, though keep in mind stuttering still occurs with minimum-possible settings (timeframes between 16-50ms consistently):
- Fullscreen (2560x1440)
- High
- vsync ON
- framecap: unlimited
- dynamic resolution: ON (using Proton's nvAPI env var, though I don't think NW utilizes it)

**Comment by kisak-valve** [comment]
New World (1063730)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5774.
@hagenmt posted on 2022-04-19T03:54:17:

# Compatibility Report
- Name of the game with compatibility issues: New World
- Steam AppID of the game: 1063730

## System Information
Distro: Ubuntu 20.04.4 LTS
Kernel: 5.13.0-39-generic
RAM:32 GB
GPU Driver:NVIDIA 510.54
GPU:NVIDIA GeForce GTX 1060 3GB
CPU:Intel Core i7-8700 @ 3.20GHz

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.



log for experimental (not bleeding edge run):
[experimental-steam-1063730.zip](https://github.com/ValveSoftware/Proton/files/8509682/experimental-steam-1063730.zip)

log for 7.0.1:
[701-steam-1063730.zip](https://github.com/ValveSoftware/Proton/files/8509680/701-steam-1063730.zip)


## Symptoms 
Tried experimental, 7.0.1, and experimental bleeding edge.  It crashes about 2-5 minutes into playing. Early on I was able to play for quite a while, but not that I have about 4-6 hours into the game it crashes every time almost instantly.  I am sitting in the wilderness at this point just collecting some ore and hemp and it crashes usually about 3-4 resources nodes into collecting every time.

I only see this other New World issue reported: https://github.com/ValveSoftware/Proton/issues/5772
But it appears to be more related to memory issues, I have 32GB of memory and I have always had a swap file as well so I don't think they are the same issue?

## Reproduction
Start steam
Launcher options: PROTON_LOG=1 bash -c 'exec "${@/NewWorldLauncher.exe/bin64\/NewWorld.exe}"' -- %command%
Hit Play
Select my character 
Search the near locations for some hemp or iron nodes and start mining
Crash after about 3-4 nodes

**Comment by kisak-valve** [comment]
Hello @hagenmt, maybe DXVK is having a vram issue with this hint:
```
err:   D3D11: Cannot create texture:
err:     Format:  97
err:     Extent:  256x512x1
err:     Samples: 1
err:     Layers:  1
err:     Levels:  1
err:     Usage:   28
err:     Flags:   0
```
---
`*** stack smashing detected ***: terminated` looks like the actual dying message of the log, could be unrelated to the comment above.

**Comment by hagenmt** [comment]
Update: I upgraded to Ubuntu 22.04 as soon as it came out few days ago.  Didn't help. When proton 7.0.2 came out though shortly after that, also didn't help but I switched to experimental bleeding edge after the 7.0.2 release and it appears the crashes are fixed!  Still having the freezes that others have mentioned maybe texture loading related (which have lead to several untimely deaths). But the hard crashes appear to be resolved.

**Comment by inittux111** [comment]
I'm experiencing crashes/freezes with the game as well, do I add my information in this issue or do I create a new issue?

**Comment by kisak-valve** [comment]
Hello @Hydranet, we're using one issue report per unofficially supported game title. This is the right place for all information related to running New World with Proton.

**Comment by inittux111** [comment]
# Compatibility Report
- Name of the game with compatibility issues: New World
- Steam AppID of the game: 1063730

## System Information
- Distribution: Arch Linux
- GPU: RX 6700XT
- Driver/LLVM version: mesa 22.0.3-1 llvm 13.0.1-2
- Kernel version: 5.17.7-arch1-1
- Link to full system information report as Gist: [System info](https://gist.githubusercontent.com/Hydranet/79ced2b43ef033c6f2debcd1cba92167/raw/ed5e90ff8ab2202922157405b5d12a0eb3794181/gistfile1.txt)
- Proton version: 7.0-2, but I have the same thing with Proton Experimental.
- Display server/Destkop Environment: Wayland/Sway
- Launch options: gamescope -ef -- gamemoderun PROTON_LOG=1  MANGOHUD=1 %command% 

## I confirm:
- [X ] that I haven't found an existing compatibility report for this game. (Adding it to this issue)
- [V ] that I have checked whether there are updates for my system available.


[steam-1063730.log.gz](https://github.com/ValveSoftware/Proton/files/8693839/steam-1063730.log.gz)

## Symptoms 
Every short while my game freezes and crashes, although I'm getting the impression it's happens more when I'm gaming and switch to my other monitor every now and then to do something on my other monitor but I'm not 100% sure.

## Reproduction
1. Launch gaming and enter server.
2.  Switch to second monitor to every 10-15 minutes
3. Switch back to my gaming monitor
4. Eventually the game freezes and crashes.

 I will do some more testing tomorrow to see what happens if I play for a few hours without switching to my other monitor. If that  gives different results I will add it as an update in this post.

**Comment by kisak-valve** [comment]
Hello @Hydranet, these look like some lines of interest from your log:
```
err:   D3D11: Cannot create texture:
err:     Format:  97
err:     Extent:  256x512x1
err:     Samples: 1
err:     Layers:  1
err:     Levels:  1
err:     Usage:   28
err:     Flags:   0
[...]
X connection to :99.0 broken (explicit kill or server shutdown).
```
Followed by an access violation (c0000005). I suspect that X (Xwayland) dying hints that there's a video driver issue to ponder.

**Comment by inittux111** [comment]
Hello @kisak-valve 
Thanks for your reply. So if I ran the game with Xorg I wouldn't be having this issue?

**Comment by Hydranet** [comment]
I played with Xorg today and it doesn't seem to be crashing then, someone I know uses Fedora 36 and Gnome Wayland session and they aren't experiencing any freezes/crashes with New World. So I installed Gnome myself and launched a Wayland session and played some New World but after a a while it freezes and crashes anyways with the same error above. Strange thing is we are using the same mesa and xwayland version.

**Comment by Hydranet** [comment]
I thought I would play again but then adding another launch option, RADV_DEBUG=hang. I though if the game would crash that might provide some info on radv, I played for about an hour and then it crashed again but no log file for radv was created.

**Comment by madrang** [comment]
Was crashing randomly too much to play, but then I disabled the in-game voice chat and after that no more crashes.

**Comment by bmjsmith** [comment]
> Was crashing randomly too much to play, but then I disabled the in-game voice chat and after that no more crashes.

Thanks - disabling this worked for me too.  Up until I tried that I kept getting crashes with:
`*** stack smashing detected ***: terminated` in the console log.

**Comment by Kuratius** [comment]
Note that the EAC support for this game seems to be semi-broken. You have to wait for it to error out and then immediately log in again without restarting the game every time you start the game.

**Comment by cglzd** [comment]
# Compatibility Report
- Name of the game with compatibility issues: New World
- Steam AppID of the game: 1063730

## System Information
- GPU: NVIDIA GeForce RTX 2080 SUPER
- Driver/LLVM version: NVIDIA 515.65.01
- Kernel version: 5.10.136-1-MANJARO
- [Gist](https://gist.github.com/carlosglez/43798a9084ebabd9dd870cdfb98b3ce4):
- Proton version: 1662670002 experimental-bleeding-edge-7.0-24419-20220908-p892322-w0d8152-d5babb2-v2120b0

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

[steam-1063730.log.zip](https://github.com/ValveSoftware/Proton/files/9531680/steam-1063730.log.zip)

## Symptoms 
I have tried all Proton version, including Proton-GE. Game runs fine in all version except for the fact that the game crashes at random time-intervals. I have noticed that whenever I join an OPR/Instance and the game crashes, I will log back on to the OPR/instance and the game wont crash for the entire duration of the OPR/instance (Doesn't matter how long the opr/instance takes) until it is over and I get ported back out.

## Reproduction
Start steam
Launcher options: PROTON_LOG=1 bash -c 'exec "${@/NewWorldLauncher.exe/bin64/NewWorld.exe}"' -- %command%
Hit Play
Select my character
Run around a little, cast a few spells, queue up for OPR. Join OPR.
If lucky I will join the OPR before game crashes, otherwise game will crash inside the OPR.

**Comment by kisak-valve** [comment]
From the log: `*** stack smashing detected ***: terminated`, which is a symptom, not a cause.

**Comment by carlosglez** [comment]
> From the log: `*** stack smashing detected ***: terminated`, which is a symptom, not a cause.

Thanks, im pretty new to linux. So I wouldn't know what that means. How do I fix it or further debug it?

**Comment by cglzd** [comment]
> From the log: `*** stack smashing detected ***: terminated`, which is a symptom, not a cause.

Thanks, im pretty new to linux. So I wouldn't know what that means. How do I fix it or further debug it?

I restarted the pc without opening anything else other than steam. Not sure if this is helpful at all but I got another log with the same issue. I also ran dmesg and got the SteamLinuxRuntime_soldier log. Let me know if there's anything else I can/should do

[steam-1063730-1.log.zip](https://github.com/ValveSoftware/Proton/files/9532097/steam-1063730-1.log.zip)

[dmesg](https://gist.github.com/carlosglez/a3373fdb7506da39c9d955a426170cf0)

[SteamLinuxRuntime_soldier](https://gist.github.com/carlosglez/e040ba1dce18b95c04a26e43fb46ded3)

Thanks :)

**Comment by cglzd** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5772#issuecomment-1241458923

Upgrading my Kernel from 5.10 LTS to 5.15 LTS resolved the issues with game crashing

**Comment by hakzsam** [comment]
Are you still able to reproduce issues with Mesa and up-to-date drivers? If so, can you explain how to reproduce? Thanks!

**Comment by Mungoid** [comment]
Not sure if it will help but disabling Steam overlay for NW seemed to fix the random crashes for me

**Comment by KernicDE** [comment]
**Compatibility Report**

    Name of the game with compatibility issues: New World
    Steam AppID of the game: 1063730

**System Information**

    Distribution: Arch Linux
    GPU: NVidia RTX 3080 FE
    Driver/LLVM version: 515.76
    Kernel version: 5.19.12-zen1-1-zen
    Proton version: ProtonGE7-36
    Display server/Destkop Environment: Wayland/Gnome
    Launch options: gamemoderun mangohud DXVK_ASYNC=1 PROTON_ENABLE_NVAPI=1 %command%

I confirm:

    [X] that I haven't found an existing compatibility report for this game. (Adding it to this issue)
    [X] that I have checked whether there are updates for my system available.

**Symptoms**

When logging in while in Windsward or porting to Windsward, then the game crashes most of the time. According to `top` the game eats up the whole memory (the free 12Gb out of 16Gb), then goes to swap and eats there 7-8Gb, then crashes. Seems like a memory leak.  
I assume, that Windsward is just the case, because it's the most populated city. First I wasn't able to login at all, as I just had 4Gb of swap. After reading here, I added another 32Gb of swap and this allows me to log in in 1 of 10 times. Steam Overlay is deactivated too.

Logging in or porting anywhere else instead of Windsward is working for me.

1. Log off in Windsward.
2. Start game.
3. Log into character.
4. See loading screen.
5. Crash back to desktop.

- I tried different Proton versions (stable, experimental, GE7-34 to GE7-36).
- I tried over several weeks, always with up-to-date system and drivers.
- I tried to repair the files.
- I removed the shader cache, no help (so starting without cache).
- I redownloaded the shader cache, no help.
- I tried with different and no Launch options.

**Comment by Apheloniush** [comment]
I have the same issue as above.
Ported to Windsward and NW started crashing so my main character is locked. I actually fed it 48gb of swap and still no go.
Replicated on Nobara Manjaro Ubuntu Endeavour and many many many proton versions official ones included
I should mention that i can log on my second char outside Windsward no problem and play for few hours
Attached a proton log as well 

[steam-1063730.log](https://github.com/ValveSoftware/Proton/files/9889495/steam-1063730.log)

**Comment by kamek-pf** [comment]
Not a proper fix as there's definitely some memory issues with this game, but setting up [zram generator](https://github.com/systemd/zram-generator) allows me to login/port to busy areas like Windsward. It would previously crash there, 9 times out of 10, and I haven't had a single crash since. 

It does swap a lot though, and the loading screen lasts for about a minute, but it doesn't crash anymore. I haven't noticed performance issues once the loading completes.

My zram config is pretty basic and adds a 16 Gb swap file. `/etc/systemd/zram-generator.conf` :

```
[zram0]
zram-size = ram
```

**Comment by FBRZGGN** [comment]
Hello,
I have the same issue as described above . I can confirm that the issue appears always during loading screens.
To bypass it I created 16Go of swap (I have 16Go of RAM) and during the first loading screen NW used about 20Go.
As @kamek-pf  said, I try to used [zram-generator](https://github.com/systemd/zram-generator), but I can make it works.
I created a question on [serverfault](https://serverfault.com/questions/1115627/zram-generator-doesnt-create-zram-device) to get help.
But even without using zram, after the first loading screen, the game is perfectly playable.

**Comment by FBRZGGN** [comment]
Hello,
I have the same issue as described by @eXeler0n above . I can confirm that the issue appears always during loading screens.
To bypass it I created 16Gb of swap (I have 16Gb of RAM) and during the first loading screen NW used about 20Gb.
As @kamek-pf  said, I try to used [zram-generator](https://github.com/systemd/zram-generator), but I can make it works.
I created a question on [serverfault](https://serverfault.com/questions/1115627/zram-generator-doesnt-create-zram-device) to get help.
But even without using zram, after the first loading screen, the game is perfectly playable.

**Comment by c6fc** [comment]
# Compatibility Report
- Name of the game with compatibility issues: New World
- Steam AppID of the game: 1063730

## System Information
- GPU: AMD Radeon RX 6900 XT
- Driver/LLVM version: 4.6 Mesa 22.0.5, LLVM 13.0.1, DRM 3.42
- Kernel version: 5.15.0-53-generic
- Link to full system information report as Gist: [SysInfo](https://gist.github.com/c6fc/adbebee43153de56e5d5d40d4c83bbb0)
- Proton version: GE-Proton7-41, Proton Experimental, Proton Next (7.0-5), and Proton 7.0-4

## I confirm:
- [ ] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

Logs for GE-Proton7-41, Proton Experimental, Proton Next (7.0-5), and Proton 7.0-4: [steam-1063730.zip](https://github.com/ValveSoftware/Proton/files/10088831/steam-1063730.zip)

## Symptoms
Splash screen opens for a few seconds, then closes. Sometimes it will disconnect my primary monitor, though this can be prevented by keeping Gnome in the 'Activity Overview' by hitting the Super (Meta) key after starting the game.

## Reproduction

1. Open the Steam library
2. Click 'New World'
3. Click 'PLAY'

**Comment by kisak-valve** [comment]
Hello @c6fc, `*** stack smashing detected ***: terminated` is the line of interest in your log.

Looking at your system information, `DRI_PRIME=pci-0000_4e_00_0` looked abnormal and could be causing mesa/RADV to fall-through to lavapipe (https://gist.github.com/c6fc/adbebee43153de56e5d5d40d4c83bbb0#file-si-txt-L177-L185). Did you set this somewhere in your environment? If you completely close Steam, then run `steam` from a terminal, does that have an effect on the game?

Separately, it looks like you're using the kisak-mesa PPA and that's up to date for OpenGL, but for some reason the mesa-vulkan-drivers system packages do not match the rest of your mesa install. This is unusual and untested, but potentially irrelevant.

**Comment by c6fc** [comment]
Thanks for the quick response. Sure enough, I had split my install over default and kisak-mesa without realizing it. I don't know where the DRI_PRIME is coming from, though the PCI address it's showing is one of the 6900 XTs. I did a bunch more testing, and here's what happened:

1. I standardized on kisak-mesa, but the computer information was showing errors about d3d12. The game crashed during the splash screen for the same four proton versions.
2. I ppa-purged kisak-mesa and went back to default 22.0.5. I noticed that it was using lavapipe as you pointed out, so I tried passing in the vulkan ICDs at the command line when running Steam. When I use x86_64, then the corresponding x11/vulkan entries for x86_64 show as using RADV drivers with no errors, while the i686 devices show: ```"vkCreateInstance (&inst_info, NULL, vk_instance) failed: ERROR_INCOMPATIBLE_DRIVER (-9)"```. Switching between them will just make the other one fail with ERROR_INCOMPATIBLE_DRIVER.
3. Using the x86_64 ICDs, I tried to start the game on all four protons, but got identical behavior.

Here's the updated [Sysinfo Gist](https://gist.github.com/c6fc/d51d5adf9e5182c3d7ed90f5c5b1997b) using the x86_64 ICDs running steam at the command line. The DRI_PRIME envvar never appeared again.

Here are the updated proton logs, using Mesa 22.0.5 with the x86_64 ICDs from the Steam command line and `PROTON_LOG=1 %command%` as the runner options: [steam-1063730_2.zip](https://github.com/ValveSoftware/Proton/files/10094330/steam-1063730_2.zip)

**Comment by c6fc** [comment]
I tried it with GE Proton 7-42 and got the same behavior. I tried using `taskset` to bind it to the first 8 cores (due to the error about >64 cores), but it didn't appear to change anything. Here's the log using the Radeon ICDs: [steam-1063730.zip](https://github.com/ValveSoftware/Proton/files/10219654/steam-1063730.zip)

Any thoughts would be appreciated.

**Comment by kisak-valve** [comment]
Hello @c6fc, give #5927 a read.

**Comment by c6fc** [comment]
`WINE_CPU_TOPOLOGY` is what I needed. I'm surprised that taskset didn't do it, but I'm up and running now, thanks for your help!

**Comment by NTMan** [comment]
Easy Anti Cheat won't let log in to the server.

[steam-1063730-proton-7.0-5.log](https://github.com/ValveSoftware/Proton/files/10254135/steam-1063730-proton-7.0-5.log)
[steam-1063730-experimental-7.0-20221208.log](https://github.com/ValveSoftware/Proton/files/10254136/steam-1063730-experimental-7.0-20221208.log)

![Screenshot from 2022-12-18 20-42-50](https://user-images.githubusercontent.com/200750/208307440-51d9a189-3e97-4c2e-bfe5-b3293b2d5056.png)

![Screenshot from 2022-12-18 20-42-53](https://user-images.githubusercontent.com/200750/208307449-4fc1582c-cbd3-48e7-9745-708fff74f64b.png)



## System Information
- GPU: AMD Radeon 6900XT
- Driver/LLVM version: [Mesa 23.0-git](https://gitlab.freedesktop.org/mesa/mesa/-/commit/58e1d14edf74e0cc2c21ddacc0aad5eb2b39a658)/15.0.6
- Kernel version: 6.2-rc0
- Link to full system information report as [gist](https://gist.github.com/NTMan/33cca4906328793206218192820779ec)
- Proton version: not matter it may be experimental-7.0-20221208 or stable proton-7.0-5

**Comment by KernicDE** [comment]
Have you installed Steam Easy-Anti cheat? You should find it in your library when searching for it.

**Comment by NTMan** [comment]
> Have you installed Steam Easy-Anti cheat? You should find it in your library when searching for it.

![Screenshot from 2022-12-19 01-58-58](https://user-images.githubusercontent.com/200750/208321527-4749b515-9ed6-42a1-be27-a0d0a48eb8a2.png)

You means Proton EasyAntiCheat Runtine? Yes it helped. But why it was not installed automatic by default?

**Comment by KernicDE** [comment]
Valve isn't checking what anti-cheat is used by every game, so they just don't know what games need EAC. And it seems, they like to have no bloat - so no EAC install without any need.

**Comment by NTMan** [comment]
Is everything bad with controller support? It seems controller support even exist, but it unusable. If move the left stick forward, then the main character in a frozen pose barely crawls forward. If you move the stick back, then the movement is much better. And this is on a usual xbox controller.

Demonstration: https://youtu.be/6bT5nOifZzE

**Comment by Kuratius** [comment]
@NTMan This isn't controller support, you're using steam's key-press PWM (pulse width modulation) to map the analog stick to keys.
New World doesn't support analog movement input. Only Amazon Games can fix this properly.

**Comment by patofet** [comment]
Hi, I cleaned installed EndeavourOS with KDE and I just installed a web browser and steam and downloaded New World and randomly when I playing all the system freezes and I don't know what to do I tried to do a ctrl + tab and tried to push the windows key to open the start menu and it doesn't do anything I have to push the power button in order to shut down the PC, what can be the problem? I miss something to install?

CPU: Intel(R) Core(TM) i9-9900K CPU @ 3.60GHz  
GPU: NVIDIA GeForce RTX 3080 Ti/PCIe/SSE2  
RAM: 32Gb
SSD: Samsung SSD 870 1Tb
Using proton experimental

I have no clue what is happening the ram usage of the last time that it freezed was around 42% without using swap

**Comment by KernicDE** [comment]
I'm not sure what causes this.
May try install gamemode (when it isn't already) and then use "gamemoderun %command%" without "" as start parameter in Steam.
Also can you check if you run on Wayland or X11? May switching to X11 when running on Wayland could help.

Edit: Also search for Proto up! in repository, install it and then install GE-Proton-xx (newest) there. Then select this instead of experimental. Hint: After installing GE you must restart Steam to see it there.

**Comment by patofet** [comment]
I'm running x11, I tried gamemoderun and proton ge and it freezed again, right now I'm testing the game with all settings at the lowest possible I also tried diferents ssd (one ssd and one nvme) and I get the same result

**Comment by joni999** [comment]
Game isn't starting anymore since last patch. Is it just me?
CPU: i7 13700k
GPU: Nvidia 3090ti
RAM: 64 GB DDR5
Proton: Proton Experimental, GE-Proton8-6, Proton8.0-2
OS: OpenSUSE Tumbleweed 20230706 & KDE Wayland
![image](https://github.com/ValveSoftware/Proton/assets/58270144/60ac2476-ff90-4f1d-b55a-db9c2a38c681)
[steam-1063730.log](https://github.com/ValveSoftware/Proton/files/11990799/steam-1063730.log)
https://gist.github.com/joni999/a0f0f7e470a9981ce30784abf7675d7d#file-steam-runtime-system-information

**Comment by inittux111** [comment]
> Game isn't starting anymore since last patch. Is it just me?
> CPU: i7 13700k
> GPU: Nvidia 3090ti
> RAM: 64 GB DDR5
> Proton: Proton Experimental, GE-Proton8-6, Proton8.0-2
> OS: OpenSUSE Tumbleweed 20230706 & KDE Wayland

I use the Flatpak version of Steam with and the community Proton version(com.valvesoftware.Steam.CompatibilityTool.Proton 7.0-5), it works for me. When I use the community ProtonGE version(com.valvesoftware.Steam.CompatibilityTool.Proton-GE) it refuses to launch. So try a different Proton version?

**Comment by joni999** [comment]
> > Game isn't starting anymore since last patch. Is it just me?
> > CPU: i7 13700k
> > GPU: Nvidia 3090ti
> > RAM: 64 GB DDR5
> > Proton: Proton Experimental, GE-Proton8-6, Proton8.0-2
> > OS: OpenSUSE Tumbleweed 20230706 & KDE Wayland
> 
> I use the Flatpak version of Steam with and the community Proton version(com.valvesoftware.Steam.CompatibilityTool.Proton 7.0-5), it works for me. When I use the community ProtonGE version(com.valvesoftware.Steam.CompatibilityTool.Proton-GE) it refuses to launch. So try a different Proton version?

Thanks - 7.0-6 works! Is there a regression in newer Proton Versions?

**Comment by kisak-valve** [comment]
Hello @joni999, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by joni999** [comment]
> Hello @joni999, please copy the contents of Steam Runtime Diagnostics from Steam (`Steam` -> `Help` -> `Steam Runtime Diagnostics`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

here's the gist:
https://gist.github.com/joni999/a0f0f7e470a9981ce30784abf7675d7d#file-steam-runtime-system-information

**Comment by kisak-valve** [comment]
New World

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6920.
@HydeFromT70s posted on 2023-07-09T15:13:16:

# Compatibility Report
- Name of the game with compatibility issues: **New World**
- Steam AppID of the game: **1063730**

## System Information
- GPU: **NVIDIA GTX 1070**
- Driver/LLVM version: **535.54.03-7 / 15.0.7-3**
- Kernel version: **6.4.2-arch1-1**
- Link to full system information report as [Gist](https://gist.github.com/HydeFromT70s/aedb7ebfb9481ad6e184ec333d9c663c)
- Proton version: **Experimental / 8.0-2**

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.

- [x] that I have checked whether there are updates for my system available.


Proton log as [Gist](https://gist.github.com/HydeFromT70s/8f818f2edbd4f6e16fcbfc9a236c2354)


## Symptoms 
The game stopped launching after the last update on July 6. In steam it has the status "Running", but after a few seconds the game crashes with an error as in the picture. Before the update the game was running without any problems.

![Crash Error](https://i.imgur.com/LjuQl5r.png)

NOTE: The game starts normal with Proton 7.0-6.

## Reproduction
Just start the game with Proton Experimental or Proton 8.0

**Comment by KorinFlakes** [comment]
Game won't even launch for me no matter what version of proton I use. Proton log has nothing written in it after the kernel line. Specs: https://gist.github.com/KorinFlakes/5bae39554652107692169a4338dcc1bc

**Comment by kisak-valve** [comment]
Hello @KorinFlakes, a header only Proton log hints that something went wrong before the game itself had a chance to run. If you completely close Steam, then run `steam-runtime` from a terminal, are there any hints in the terminal spew when you tell Steam to start the game?

**Comment by KorinFlakes** [comment]
Thank you, there was a problem with the compatdata/1063730 folder so I deleted that and re-ran the game and it launched.

**Comment by kisak-valve** [comment]
New world crashing very often during loading screens

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7234.
@Kitsab77 posted on 2023-11-04T22:32:48:

# Compatibility Report
- Name of the game with compatibility issues: New world
- Steam AppID of the game: https://store.steampowered.com/app/1063730/New_World/ -> 1063730

## System Information

- GPU: AMD (Navi33) 7600s
- Video driver version: amdgpu 23.10
- Kernel version: 6.5.5-1-MANJARO
- Link to full system information report as [Gist](https://gist.github.com/): https://hastebin.com/share/mazenaqevu.yaml
- Proton version: Proton Experimental (also tried 8.0-4, 7.0-6, 5.13-6, and Proton-GE which is a custom build)

## I confirm:
- [ x] that I haven't found an existing compatibility report for this game **relating to this error**.
- [ x] that I have checked whether there are updates for my system available. **pacman -syu / yay -Sua**

 https://drive.google.com/file/d/1LVnKnSszH5Usy1HA-ZPTzLCaj5I76WrZ/view?usp=sharing
Game always crashes with a bunch of:
err:sync:RtlpWaitForCriticalSection section 000000017006F2A0 "../src-wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 0340, blocked by 0784, retrying (60 sec)
at the end.

## Symptoms 
The game is crashing during loading screen (mostly in bigger cities or crowded places)
The crash always happens during loading screen. When logging into the game it needs sometimes up to 10 tries to logon.
Also happening when using recall to tavern or fast travel portals to cities. Sometimes also happens outside of cities.
Tried mesa drivers, tried amdgpu driver installed from Archlinux[Aur], tried different wine versions.
Actual launch options:
PROTON_LOG=1 DRI_PRIME=1 %command% -dx12 -high -novid -nojoy -vulkan -disable_d3d9ex -nohltv
Basic launch options I tried: 
DRI_PRIME=1 %command% -dx12 -high

## Reproduction
Could be reproduced very well, happening everytime I play the game. 

I searched the web for err:sync:RtlpWaitForCriticalSection, I found this should have been fixed by release Wine6.1

I tried a lot chaning graphic details, but this seems to be replated to a internet/game server connection timeout issue, or may I misunderstand this?

Thanks and best regards kitsab

**Comment by Kitsab77** [comment]
> New world crashing very often during loading screens

[Edit, did not solve the issue - it happened again]

Hello,

possibly I found something after long repeating duckduckgo - searches.
There was a message board post relating to this issue 
https://www.reddit.com/r/linux_gaming/comments/143tjqb/tales_of_arise_overloading_ryzen_5600x_when/
They advice to modify this:
sysctl vm.max_map_count
vm.max_map_count = 65530
to 2147483642
with the command
sudo sysctl -w vm.max_map_count=2147483642
This is a system memory handler argument from my understanding.
The discussion refers to this information:
https://www.phoronix.com/news/Fedora-39-VM-Max-Map-Count

I had the issue trying to logon ~10 times again, after modifying this value it worked instandly - hopefully this solves the problem.

Best regards

Kitsab

**Comment by kisak-valve** [comment]
New World Crashing on Startup

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7341.
@bigwheel50 posted on 2023-12-13T18:31:14:

# Compatibility Report
- Name of the game with compatibility issues: New World
- Steam AppID of the game: 1063730

## System Information
- GPU: Nvidia RTX 4090
- Video driver version: Nvidia 545.29.06
- Kernel version: 545.29.06
- Link to full system information report as [Gist](https://gist.github.com/bigwheel50/9387dd7d7460814ca43f6b0fa78a766f):
- Proton version: 8.0-4, Experimental, GE-Proton8-25, GE-Proton8-20, GE-Proton8-6, GE-Proton7-16. GE-Proton7-14

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

[steam-1063730.zip](https://github.com/ValveSoftware/Proton/files/13664092/steam-1063730.zip)

## Symptoms
The game crashes immediately during startup. 

## Reproduction

1. Launch game
2. [Don't] Profit

Have tried multiple versions of Proton, GE-Proton, launch options, etc.

**Comment by kisak-valve** [comment]
Hello @bigwheel50, looking at your system information, it looks like Vulkan is broken in general on your system (https://gist.github.com/bigwheel50/9387dd7d7460814ca43f6b0fa78a766f#file-gistfile1-txt-L3573-L3584) I suspect this is from something setting `DRI_PRIME=pci-0000_5c_00_0` (https://gist.github.com/bigwheel50/9387dd7d7460814ca43f6b0fa78a766f#file-gistfile1-txt-L153) which is then breaking the Vulkan render path.

Blind guess, if you completely close Steam, then run `steam` from a terminal, does that have an effect on that environment variable getting set and the game's behavior?

**Comment by bigwheel50** [comment]
Thank you for the quick reply! Launching steam through the terminal does seem to impact the `DRI_PRIME` code (it no longer appears at all in the runtime diagnostic output), however, Vulkan seems to remain the same, as does the crashing issue.

**Comment by stiangrindvoll** [comment]
With New World's season 4 update it started to crash for me as well after the loading screen.
On Proton v7.0-6 that i have been using, it gives me this error, dont have any more logs than this for now. 
![Screenshot from 2023-12-13 19-21-04](https://github.com/ValveSoftware/Proton/assets/3809634/3dbfebf4-c898-4d77-b21f-784eea155e04)

**Comment by bigwheel50** [comment]
I've gotten that same error when trying older versions of Proton/GE-Proton as well. 

As @stiangrindvoll mentioned, the issue started this morning after a large patch. Up until this, things were running smoothly.

**Comment by stiangrindvoll** [comment]
Here is my proton steam log file
[steam-1063730.log.gz](https://github.com/ValveSoftware/Proton/files/13664670/steam-1063730.log.gz)

**Comment by Blisto91** [comment]
I assume you both have high core count CPUs.
Can you check if it works by limiting the amount of logical cores to 26? 
`WINE_CPU_TOPOLOGY=26:0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25 %command%`
Or even lower if that doesn't help.

Edit: Playing with 16 seems stable so far

**Comment by stiangrindvoll** [comment]
Yes this worked! Thank you @Blisto91 im currently testing with 18.

**Comment by bigwheel50** [comment]
Wooo! Can confirm that worked for me as well! 26 still failed, however, I tried the aforementioned 18 count and that did the trick!

**Comment by Blisto91** [comment]
Cool thanks. 26 seemed to do the trick for me, but might be a bit random. Shouldn't hurt the game either way.

Edit: actually 26 could still crash ingame at times. 16 seems stable so far

**Comment by stiangrindvoll** [comment]
i had crashes at 18 too, randomly while in the game. At 12, i've had no single crash yet.

**Comment by mailhotz** [comment]
New World eventually hard crashes my machine and have to hard reboot.

# Compatibility Report
- Name of the game with compatibility issues: New World
- Steam AppID of the game: 1063730

## System Information
![Screenshot from 2023-12-20 19-10-31](https://github.com/ValveSoftware/Proton/assets/5949142/b0261ac1-639d-47f3-ae73-0848aefe152b)
- [Gist](https://gist.github.com/mailhotz/11ae387d0e6ff4c82691828b4490962b)
- Proton Version: Experimental but have tried many different versions

# I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1063730.zip](https://github.com/ValveSoftware/Proton/files/13733763/steam-1063730.zip)

# Symptoms:
Game loads in fine, occasionally crashes after character select or sporadically during gameplay. It may be of note that my journal gets flooded with the following log line, on average it's posting the overrun log with different values about every 2-3ms
```
localhost.localdomain pipewire-pulse[2847]: mod.protocol-pulse: 0x55a26846aaa0: [New World] overrun recover read:230400 avail:17408 max:15360 skip:13568
```
Example of frequency:
```
Dec 20 19:00:34 localhost.localdomain pipewire-pulse[2847]: mod.protocol-pulse: 0x55a26846aaa0: [New World] overrun recover read:41568768 avail:28160 max:15360 skip:24320
Dec 20 19:00:35 localhost.localdomain pipewire-pulse[2847]: mod.protocol-pulse: 0x55a26846aaa0: [New World] overrun recover read:41846528 avail:16640 max:15360 skip:12800
Dec 20 19:00:35 localhost.localdomain pipewire-pulse[2847]: mod.protocol-pulse: 0x55a26a998b50: [New World] overrun recover read:40660480 avail:16896 max:15360 skip:13056
Dec 20 19:00:39 localhost.localdomain pipewire-pulse[2847]: mod.protocol-pulse: 0x55a26a998b50: [New World] overrun recover read:42370816 avail:16640 max:15360 skip:12800
Dec 20 19:00:42 localhost.localdomain pipewire-pulse[2847]: mod.protocol-pulse: 0x55a26a998b50: [New World] overrun recover read:43474176 avail:17152 max:15360 skip:13312
```

Do not seem to have this log appear in any game besides New World.

[Game crash in journal after freezing machine](https://gist.github.com/mailhotz/c883e25f2ef618c3976fccd6f2b5bba0)

# Reproduction:
- Launch game from steam with `journalctl -f`
- Once selecting character and load into world
- Begin to see the above pipewire-pulse spew until game is closed or crashes OS

**Comment by Unaccounted4** [comment]
I have a similar issue to what @mailhotz is reporting. On my end it doesn't crash the entire OS but the game itself will crash after 2 hours of continuous play. Checking journalctl reveals it spews that message several times per second.

System specs: https://gist.github.com/Unaccounted4/2e1ee0dd64ccc5c6801740e3cbe4ab89

Sample of 2 seconds in journalctl:
```
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:148922368 avail:16384 max:15360 skip:12544
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:149659136 avail:25088 max:15360 skip:21248
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161242624 avail:41472 max:15360 skip:37632
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:149684224 avail:16384 max:15360 skip:12544
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:148938752 avail:32768 max:15360 skip:28928
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161337856 avail:19968 max:15360 skip:16128
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:149754368 avail:19968 max:15360 skip:16128
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149017600 avail:27648 max:15360 skip:23808
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161380864 avail:17920 max:15360 skip:14080
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:149797376 avail:17920 max:15360 skip:14080
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149060608 avail:33792 max:15360 skip:29952
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:149853696 avail:27136 max:15360 skip:23296
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149117440 avail:34304 max:15360 skip:30464
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161479424 avail:17664 max:15360 skip:13824
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:149896192 avail:25600 max:15360 skip:21760
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149151744 avail:40960 max:15360 skip:37120
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161512448 avail:17408 max:15360 skip:13568
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:149929472 avail:25088 max:15360 skip:21248
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149192704 avail:32768 max:15360 skip:28928
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161568256 avail:18944 max:15360 skip:15104
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353bf77ab0: [New World] overrun recover read:246136576 avail:24832 max:15360 skip:20992
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161587200 avail:16384 max:15360 skip:12544
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149240832 avail:33792 max:15360 skip:29952
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:149977600 avail:42496 max:15360 skip:38656
jan 29 22:35:00 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149274624 avail:16384 max:15360 skip:12544
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149298688 avail:16896 max:15360 skip:13056
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161641984 avail:18944 max:15360 skip:15104
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:150027776 avail:49664 max:15360 skip:45824
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149323264 avail:25088 max:15360 skip:21248
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149348352 avail:16384 max:15360 skip:12544
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353bf77ab0: [New World] overrun recover read:246257408 avail:18688 max:15360 skip:14848
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161691648 avail:26624 max:15360 skip:22784
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:150108160 avail:26624 max:15360 skip:22784
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb4b890: [New World] overrun recover read:161718272 avail:16384 max:15360 skip:12544
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:150134784 avail:16384 max:15360 skip:12544
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149380096 avail:41984 max:15360 skip:38144
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353cb3c090: [New World] overrun recover read:150158848 avail:16896 max:15360 skip:13056
jan 29 22:35:01 Desktop pipewire-pulse[4597]: mod.protocol-pulse: 0x56353a5daa10: [New World] overrun recover read:149422080 avail:24576 max:15360 skip:20736
```

**Comment by AFrancq** [comment]
Repeatedly crashing in game after 15-40 minutes of gameplay. Happens in a variety of locations.

Startup settings: 
`ENABLE_VKBASALT=1 RADV_PERFTEST=gpl YMMV_VULKAN_ICD=RADV  WINE_CPU_TOPOLOGY=8:0,1,2,3,4,5,6,7 %command% > ~/NWout.log 2> ~/NWerr.log`

Here is the ourput of NWerr.log:
```ERROR: ld.so: object '/home/ash/.local/share/Steam/ubuntu12_64/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
ERROR: ld.so: object '/home/ash/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/ash/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/ash/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
x86_64-linux-gnu-capsule-capture-libs: warning: Dependencies of libnvidia-pkcs11.so.550.78 not found, ignoring: Missing dependencies: Could not find "libcrypto.so.1.1" in LD_LIBRARY_PATH "/usr/lib/x86_64-linux-gnu/libfakeroot:/lib/i386-linux-gnu:/usr/local/lib:/lib/x86_64-linux-gnu:/lib:/home/ash/.local/share/Steam/steamapps/common/New World", ld.so.cache, DT_RUNPATH or fallback /lib:/usr/lib
esync: up and running.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
Setting breakpad minidump AppID = 1063730
Steam_SetMinidumpSteamID:  Caching Steam ID:  76561198055466734 [API loaded no]
[356:360:20240429,123858.830:WARNING registration_protocol_win.cc:103] SendToCrashHandlerServer(): CreateFile("000000000376CCD0") failed.: File not found. (2)
Fossilize INFO: Overriding serialization path: "/home/ash/.local/share/Steam/steamapps/shadercache/1063730/fozpipelinesv6/steamapprun_pipeline_cache".
Setting breakpad minidump AppID = 1063730
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  76561198055466734 [API loaded no]
Fossilize INFO: Overriding serialization path: "/home/ash/.local/share/Steam/steamapps/shadercache/1063730/fozpipelinesv6/steamapprun_pipeline_cache".
Fossilize INFO: Overriding serialization path: "/home/ash/.local/share/Steam/steamapps/shadercache/1063730/fozpipelinesv6/steamapprun_pipeline_cache".
*** stack smashing detected ***: terminated
pid 5923 != 5922, skipping destruction (fork without exec?)
```


The error log shows up to "Fossilize INFO" while starting and then the crash is always "stack smashing detected". I've tried this on Proton 7, 8, 9 beta, and experimental through Steam's compatibility tool.

Does anyone have any idea on next steps? I keep crashing in combat and dying.

**Comment by AFrancq** [comment]
The game seems to have a problem with multi-threading and proton, I have been able to achieve stable performance (albeit not great performance) with the following change to my launch options: `WINE_CPU_TOPOLOGY=4:0,2,4,6`

EDIT: @blae-s why thumbs down?

**Comment by Milor123** [comment]
Someone solution for the crashes? I have a rtx 4070 and a i5 13500, i am using proton 9-10 and Manjaro with wayland

**Comment by kisak-valve** [comment]
New World: Aeternum (1063730)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/8182.
@kosimas posted on 2024-10-20T15:05:08:

# Compatibility Report
- Name of the game with compatibility issues: New World: Aeternum
- Steam AppID of the game: 1063730

## System Information
- GPU: AMD Radeon RX 7900 XTX
- Video driver version: Mesa 24.2.4-arch1.1
- Kernel version: 6.11.3-arch1-1
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/kosimas/9c615b77c06d28f10e1e1d5c7a95bd01
- Proton version: 9.X, Experimental, GE Proton 9.X
[steam-1063730.log](https://github.com/user-attachments/files/17450753/steam-1063730.log)


## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.



## Symptoms 
Before the game launches I get a Window with the following contents: Unsupported CPU detected: CPU needs to support SSE, SSE2 and SSE3.
When clicking okay in that window the game launches without any problems but only 4 cores of my 32 cores are being used when playing. This results in poor performance.

## Reproduction
To reproduce the problem someone probably needs to own at least a 13th gen Intel CPU.

**Comment by kosimas** [comment]
The "Unsupported CPU detected" issue was caused by gamemode. Without using gamemode everything works as expected and without any performance issues.

**Comment by Unaccounted4** [comment]
Steam Overlay doesn't seem to be working on my PC, which breaks Steam Input and the in-game cash store. Honestly that last one may be a positive, but I'm sure AGS disagrees.

I've tried with and without gamescope, it makes no difference. Adding or removing the -e flag in gamescope also doesn't make a difference.

The issue seems to be the EAC pop-up, as I can see the little "Shift+Tab to open" toast notification on the EAC pop-up. Then nothing in game proper.
